industrial_ci/src/util.sh

```diff
@@ -136,3 +136,9 @@ function error {
     fi
     ici_exit $exit_code
 }
+
+if ! which sudo > /dev/null; then
```

Comment:
This portion seems like another magic of `(ba)sh` to me, and apparently there's nothing wrong nor confusing. It's just...impressive.

Reply:
And it turned out to be wrong [somehow](https://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script).. (but works for us)
We should migrate it to `command -v` on the next occasion